src/components/cc-pricing-estimation/cc-pricing-estimation.js

```diff
@@ -1,23 +1,77 @@
 import '../cc-button/cc-button.js';
-import '../cc-error/cc-error.js';
-import '../cc-input-number/cc-input-number.js';
+import '../cc-icon/cc-icon.js';
+import '../cc-badge/cc-badge.js';
+import '../cc-expand/cc-expand.js';
 import { css, html, LitElement } from 'lit';
+import { classMap } from 'lit/directives/class-map.js';
+import { createRef, ref } from 'lit/directives/ref.js';
+import {
+  iconRemixDeleteBin_4Line as iconBin,
+  iconRemixArrowDownSLine as iconArrowDown,
+  iconRemixAddLine as iconAdd,
+  iconRemixSubtractLine as iconSubtract,
+} from '../../assets/cc-remix.icons.js';
+import { LostFocusController } from '../../controllers/lost-focus-controller.js';
 import { dispatchCustomEvent } from '../../lib/events.js';
 import { i18n } from '../../lib/i18n.js';
-import { withResizeObserver } from '../../mixins/with-resize-observer/with-resize-observer.js';
-import { ccLink } from '../../templates/cc-link/cc-link.js';
-
-const deleteSvg = new URL('../../assets/delete.svg', import.meta.url).href;
+import '@shoelace-style/shoelace/dist/components/select/select.js';
+import '@shoelace-style/shoelace/dist/components/option/option.js';
+import { accessibilityStyles } from '../../styles/accessibility.js';
+import { shoelaceStyles } from '../../styles/shoelace.js';
+
+const FEATURES_I18N = {
+  'connection-limit': () => i18n('cc-pricing-estimation.feature.connection-limit'),
+  cpu: () => i18n('cc-pricing-estimation.feature.cpu'),
+  databases: () => i18n('cc-pricing-estimation.feature.databases'),
+  'disk-size': () => i18n('cc-pricing-estimation.feature.disk-size'),
+  gpu: () => i18n('cc-pricing-estimation.feature.gpu'),
+  'has-logs': () => i18n('cc-pricing-estimation.feature.has-logs'),
+  'has-metrics': () => i18n('cc-pricing-estimation.feature.has-metrics'),
+  'max-db-size': () => i18n('cc-pricing-estimation.feature.max-db-size'),
+  memory: () => i18n('cc-pricing-estimation.feature.memory'),
+  version: () => i18n('cc-pricing-estimation.feature.version'),
+};
+const AVAILABLE_FEATURES = Object.keys(FEATURES_I18N);
 
 /** @type {Currency} */
-const CURRENCY_EUR = { code: 'EUR', changeRate: 1 };
-
-const CONTACT_URL = 'https://www.clever-cloud.com/en/contact-sales';
-const SIGN_UP_URL = 'https://api.clever-cloud.com/v2/sessions/signup';
+const DEFAULT_CURRENCY = { code: 'EUR', changeRate: 1 };
+
+/** @type {Currency[]} */
+const DEFAULT_CURRENCY_LIST = [
+  { code: 'EUR', changeRate: 1 },
+  { code: 'GBP', changeRate: 0.88603 },
+  { code: 'USD', changeRate: 1.2091 },
```

Comment:
issue: I'm not sure the component should have a hard coded list of currencies with changeRates.

I would remove this const and do this in the constructor:

```js
this.currencyList = [DEFAULT_CURRENCY];
```